src/Command/DebugMappingCommand.php

```diff
@@ -0,0 +1,58 @@
+<?php
+declare(strict_types=1);
+
+namespace League\Tactician\Bundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+class DebugMappingCommand extends ContainerAwareCommand
```

Comment:
It should extend `Symfony\Component\Console\Command\Command`, no need for being container aware